pkg/capabilities/capabilities.go

```diff
@@ -127,9 +127,20 @@ type BaseCapability interface {
 	Info(ctx context.Context) (CapabilityInfo, error)
 }
 
+type TriggerRegistrationRequest struct {
+	TriggerID string
```

Comment:
If this is made unique per workflow by the engine, could we indicate it with a doc comment?